### ISSUE TITLE
Mes données locales semblent obsolètes par rapport aux dernières mises à jour

### ISSUE THREAD

**Issue by mlbiche** [datagir/nosgestesclimat-site#842]
> ⚠️ Avant de créer une issue, lisez attentivement [notre FAQ](https://nosgestesclimat.fr/contribuer) qui explicite un bon nombre des limites du test nosgestesclimat.fr.

## Poser une question 

En accédant de nouveau à [nosgestesclimat.fr/profil](https://nosgestesclimat.fr/profil) après avoir mis à jour mon questionnaire aujourd'hui, je ne peux plus ouvrir la catégorie Alimentation à cause de règles inconnues.

En détail, voilà toutes les règles qui sont détectées comme inconnues dans mes données locales :
* `alimentation . déchets . niveau d'engagement`
* `alimentation . gaspillage alimentaire . fréquence`
* `divers . électroménager . réfrigérateur . présent`
* `divers . électroménager . mini réfrigérateur . présent`
* `divers . électroménager . lave-linge . présent`
* `divers . électroménager . sèche-linge . présent`
* `divers . électroménager . lave-vaisselle . présent`
* `divers . électroménager . four . présent`
* `divers . électroménager . micro-ondes . présent`
* `divers . électroménager . plaques . présent`
* `divers . électroménager . bouilloire . présent`
* `divers . électroménager . cafetière . présent`
* `divers . électroménager . aspirateur . présent`
* `divers . électroménager . robot cuisine . présent`
* `divers . électroménager . tondeuse électrique . présent`
* `numérique . préservation`
* `numérique . appareil photo . présent`
* `numérique . home cinéma . présent`
* `numérique . ordinateur portable . présent`
* `numérique . ordinateur fixe . présent`
* `numérique . téléphone . présent`
* `numérique . tablette . présent`
* `numérique . TV . présent`
* `numérique . vidéoprojecteur . présent`
* `numérique . enceinte bluetooth . présent`
* `numérique . enceinte vocale . présent`
* `numérique . montre connectée . présent`
* `numérique . console de salon . présent`
* `numérique . console portable . présent`
* `numérique . internet . durée journalière`
* `numérique . téléphone . type`
* `services publics . question rhétorique`
* `alimentation . petit déjeuner . type . français` - Cette dernière règle est la seule qui fait planter l'application (écran blanc) et qui a lieu suite à une interaction (déplier la catégorie "Alimentation").

La plupart correspondent à des questions qui m'ont été posées aujourd'hui (reposées j'ai l'impression ?). Seule la question sur le petit-déjeuner ne m'a pas été reposée.

Les options sur l'électroménager et le numérique ont été cochées ou non (je peux vous donner le détail si ça vous aide).

## Proposer une amélioration 

Il semblerait qu'il y ait un problème de rétro-compatibilité avec ma donnée locale. J'aurais bien filé un coup de main là-dessus mais je ne sais pas quelle politique vous souhaitez appliquer vis-à-vis de ce problème.

**Comment by mlbiche** [comment]
Effectivement, parmi toutes les règles que j'ai mentionnées dans le ticket, c'est la seule qui ne m'ait pas été reposée ce jour-là. Et donc la seule qui entrainait un crash de l'application depuis la page Profil.

**Comment by Clemog** [comment]
@florianpanchout c'est le pb dont je te parlais qui est lié aux tests que tu as pu faire sur les personas :)
